src/edu/csus/ecs/pc2/core/execute/SandboxUtilities.java

```diff
@@ -10,7 +10,7 @@
 import edu.csus.ecs.pc2.core.model.Run;
 
 /**
- * 
+ * Input to Sandbox. 
  * 
```

Comment:
It seems inaccurate that the comment here is "Input to Sandbox" when the class is "SandboxUtilities".  A "Utilities" class is in general quite different from something which provides input.  What if the Utilities are later expanded to produce output, for example?

Reply:
Will remove comment and add doc later, added TODO